packages/angular/src/lib/browser/index.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  OutputRef,
+  Provider,
+  provideZonelessChangeDetection,
+  Type,
+} from '@angular/core';
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import {
+  BrowserTestingModule,
+  platformBrowserTesting,
+} from '@angular/platform-browser/testing';
+import {
+  BrowserMount,
+  OUTPUT_BUS_VARIABLE_NAME,
+  OutputEvent,
+  OutputTypes,
+} from '../common';
+import { TestronautAngularNotSetUpError } from '../playwright/testronaut-angular-not-set-up.error';
+import { getComponentOutputs } from './ng-internals';
+
+export const mount = async <CMP_TYPE extends Type<unknown>>(
+  ...args: Parameters<BrowserMount<CMP_TYPE>>
+): ReturnType<BrowserMount<CMP_TYPE>> => {
+  const [cmp, { inputs = {} } = {}] = args;
+
+  assertIsSetUp();
+  const g = globalThis as unknown as {
+    [OUTPUT_BUS_VARIABLE_NAME]: (
+      outputEvent: OutputEvent<InstanceType<typeof cmp>>
+    ) => void;
+  };
+
+  const fixture = TestBed.createComponent(cmp) as ComponentFixture<
+    InstanceType<CMP_TYPE>
+  >;
+
+  Object.entries(inputs).forEach(([key, value]) => {
+    fixture.componentRef.setInput(key, value);
+  });
+
+  await fixture.whenStable();
+
+  const outputNames = getComponentOutputs(cmp);
+
+  for (const outputName of outputNames) {
+    subscribeToOutput(fixture.componentInstance, outputName, (value) => {
+      g[OUTPUT_BUS_VARIABLE_NAME]({
+        outputName,
+        value,
+      });
+    });
+  }
+
+  return { outputNames };
+};
+
+export async function configure({ providers }: { providers: Provider[] }) {
+  assertIsSetUp();
+
+  TestBed.configureTestingModule({ providers });
+}
+
+export async function setUpTestronautAngular() {
+  TestBed.initTestEnvironment(BrowserTestingModule, platformBrowserTesting());
+
+  TestBed.configureTestingModule({
+    providers: [provideZonelessChangeDetection()],
+  });
+
+  isSetUp = true;
+}
+
+function assertIsSetUp() {
+  if (!isSetUp) {
+    throw new TestronautAngularNotSetUpError();
+  }
+}
+
+function subscribeToOutput<CMP, PROP extends keyof OutputTypes<CMP>>(
+  componentInstance: CMP,
+  outputName: PROP,
+  callback: (value: OutputTypes<CMP>[PROP]) => void
+) {
+  (componentInstance[outputName] as OutputRef<CMP[PROP]>)?.subscribe(
```

Comment:
I don't want to clutter the code with subscription management here. What do you think?
The only edge case is `@Output() myOutput = source$;` where source$ is an observable with side effects etc... + user is reusing the same page + the side-effect is changing something global.

`outputFromObservable(source$)` users are safe because this one will unsubscribe automatically from the upstream `source$.`